_extras/training_calendar.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: page
+title: "Instructor Training Calendar for Carpentries Partners"
+permalink: /training_calendar/
+---
+
+
+#### Carpentries Instructor Training Calendar
+###  September - December 2017
+
+
+If you are a partner site ready to schedule your instructor training session, please contact maneesha@carpentries.org with your first and second choices for dates.
+
+All trainings are offered online from 9 am to 5 pm in the time zone noted.  You do not need to be in the same time zone yourself but need to accommodate that time zone. 
+
+Please keep in mind Daylight savings time changes on November 5, 2017.
```

Comment:
Daylight saving change in the US and Europe are different. Europe will change on Sunday, 29 October. And South Hemisphere countries will start Daylight saving in a different time. I recommend change this line to

> Please keep in mind Daylight savings time changes on the time zone noted.